terminal_colors/terminal_colors.py

```diff
@@ -47,6 +47,12 @@
         - use of color intensity to determine foreground contrast color
         - 16 color SGR ANSI chart
         - 88 color rgb display bugfix
+    Mike Clark <mike.clark.8192@gmail.com> - support for Windows:
+        - declared dependency on windows-curses (only when installing on Windows)
+        - added --force-term N to test for color support outside of tigetnum('colors')
+        - rely on PyPA entry_points= to setup cross-platform `terminal-colors` command,
+          (which required renaming the script to make it addressable as a module)
```

Comment:
```suggestion
```
I realize now that this is not the place for this comment, since this script is sometimes used outside the context of pip or a python package.